lms/templates/admin/macros.html.jinja2

```diff
@@ -115,17 +116,29 @@
 {% endmacro %}
 
 
-{% macro instances_table(request,  instances) %}
+{% macro instances_table(request, instances) %}
 {{ object_list_table(
     request, 'admin.instance', instances,
-    ['Name', 'Consumer key', 'GUID', 'Deployment ID'],
-    ['name', 'consumer_key', 'tool_consumer_instance_guid', 'deployment_id']
+    fields=[
+        {"label": "Name", "name": "name"},
+        {"label": "Created", "name": "created"},
+        {"label": "Last Launched", "name": "last_launched"},
+        {"label": "Product Family", "name": "tool_consumer_info_product_family_code"},
+        {"label": "Email", "name": "requesters_email"},
+        {"label": "LMS URL", "name": "lms_url"},
+    ]
```

Comment:
Originally I made this change with the idea of adding an optional "format" argument on the end with a function and passing in a function or something similar. We might yet do that, but we don't in this PR.

I actually think this way of doing things has some advantages. It's easier to see which fields and names are correlated and it's much easier to re-order the columns if you want to.